app/views/modals/set-home-page-modal.html

```diff
@@ -0,0 +1,50 @@
+<div class="modal-resource-action">
+  <div class="modal-header">
+    <h1>Set Home Page</h1>
+  </div>
+  <div class="modal-body indent">
+    <p>
+      <div class="title">Set your preferred home page.</div>
+      Note: This setting is browser specific and will not be maintained across browsers.
+    </p>
+    <ng-form name="form">
```

Comment:
```html
<form name="form">
```

Give it a more specific name, though. Since this is a dialog shown on any page, the generic name `form` might already be used.

Reply:
Fixed